test/extended/etcd/etcd_storage_path.go

```diff
@@ -334,11 +322,18 @@ func testEtcd3StoragePath(t g.GinkgoTInterface, oc *exutil.CLI, etcdClient3Fn fu
 
 		// Removed etcd data.
 
-		// disabled deprecated versions in 1.26
 		removeStorageData(t, etcdStorageData,
+			// disabled deprecated versions in 1.26
 			gvr("autoscaling", "v2beta2", "horizontalpodautoscalers"),
 			gvr("flowcontrol.apiserver.k8s.io", "v1beta1", "flowschemas"),
 			gvr("flowcontrol.apiserver.k8s.io", "v1beta1", "prioritylevelconfigurations"),
+			// disabled alpha versions in 1.26
+			gvr("resource.k8s.io", "v1alpha1", "resourceclasses"),
+			gvr("admissionregistration.k8s.io", "v1alpha1", "validatingadmissionpolicybindings"),
+			gvr("admissionregistration.k8s.io", "v1alpha1", "validatingadmissionpolicies"),
+			gvr("resource.k8s.io", "v1alpha1", "resourceclaimtemplates"),
+			gvr("resource.k8s.io", "v1alpha1", "resourceclaims"),
+			gvr("resource.k8s.io", "v1alpha1", "podschedulings"),
```

Comment:
Same for the above "modified" section, since we're reading what we're getting from k8s 1.26. Ideally this whole section will be empty and prepared for 1.27 bump, like so:
```golang
	// Apply output of git diff origin/release-1.XY origin/release-1.X(Y+1) test/integration/etcd/data.go. This is needed
	// to apply the right data depending on the kube version of the running server. Replace this with the next current
	// and rebase version next time. Don't pile them up.
	if strings.HasPrefix(version.Minor, "27") {
		for k, a := range map[schema.GroupVersionResource]etcddata.StorageData{
			// Added etcd data.
			// TODO: fill when rebase has started
		} {
			if _, preexisting := etcdStorageData[k]; preexisting {
				t.Errorf("upstream etcd storage data already has data for %v. Update current and rebase version diff to next rebase version", k)
			}
			etcdStorageData[k] = a
		}

		// Modified etcd data.
		// TODO: fill when rebase has started

		// Removed etcd data.
		// TODO: fill when rebase has started
		removeStorageData(t, etcdStorageData)
	}
```

Reply:
Done.